curriculum/Chapter-?-Databases.md

```diff
@@ -0,0 +1,30 @@
+# Chapter ?: Databases
+
+## Objectives
+
+After completing this chapter, you will be able to:
+
+- Understand the role and importance of databases in application development.
+- Differentiate between graph, document, and relational-based databases.
+- Implement basic CRUD operations using MongoDB in a NestJS project.
+- Appreciate the nuances of selecting a database type for specific application needs.
+
+## Understanding Database Types
+
+Databases can broadly be categorized based on the data model they use:
```

Comment:
There are three main types of databases: Column Relational (SQL), Document, and Node Relational (Graph).